spec/api/made_to_measure/messages_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe 'MadeToMeasure::Messages' do
+  context 'v0.1' do
```

Comment:
well, we're gonna have a v1 at some point, I guess I'd just argue we should make this v1 instead of 0.1